models/articles.model.js

```diff
@@ -18,4 +18,20 @@ exports.selectArticleById = (article_id) => {
         }
         return result.rows[0];
     })
-} 
\ No newline at end of file
+} 
+
+exports.selectArticleComment = (article_id) => {
+    return db.query(`SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id from comments JOIN articles ON comments.article_id = articles.article_id WHERE articles.article_id = $1 ORDER BY comments.created_at DESC;`, [article_id])
+    .then((result) => {
+        return result.rows
+    })
+}
+
+exports.checkArticleExists = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({rows}) => {
+        if(rows.length === 0){
+            return Promise.reject({status:404, msg:"Article not found"})
+        }
+    })
+}
```

Comment:
While this is completely fine, Is it not just an exact copy of the select ArticleById model?